R/utils.R

```diff
@@ -173,9 +173,14 @@ write_pbp <- function(seasons, dbConnection, tablename){
   purrr::walk(seasons, function(x, p){
     pbp <- nflreadr::load_pbp(x)
     if (!DBI::dbExistsTable(dbConnection, tablename)){
+      default_play$weather <- "some other string"
```

Comment:
Would prefer to see this fixed here:  https://github.com/nflverse/nflfastR/blob/master/data-raw/default_play.R   rather than a one liner fix in the code

Reply:
Yes, I also think that this file would be the natural way to fix it. But this files starts with
### Create datatype dataframe
### This is a db that is stored on Seb's local machine
So I can't fix the problem there. Maybe @mrcaseb can have a look at this.

I think the main reason for this bug is that we hope DBI::dbWriteTable() will create the right column types based on `default_play` (and the first season uploaded).
So if I can't change `default_play` as mentioned above I suggest loading the seasons in reverse order as an alternative solution.

Reply:
It's indeed a local db. However, the local db is just the result of a "fresh" `update_db()` run. 

We have added the default play because we were having issues with MySQL data bases and quite strict variable definitions, esp. for strings.

Reply:
Looks like the problem is that I used the 1999 season to create the default play.